src/public_key.rs

```diff
@@ -282,7 +282,7 @@ mod tests {
     use std::collections::hash_map::DefaultHasher;
     use std::hash::{Hash, Hasher};
 
-    fn default_bytes() -> [u8; 33] {
+    const fn default_bytes() -> [u8; 33] {
```

Comment:
You don't even need a function for default bytes!

Reply:
lol. airplane brain